### DISCUSSION TITLE
Add option to allow tooltip to be interactive (a la tippy)

### DISCUSSION THREAD

**Discussion by tnrich** [palantir/blueprint#5749] [Q&A] [answered]
Hey there,

I think it would be great to add an "isInteractive" flag to <Tooltip> to allow the content inside them to be interacted with. 

Here's an example from the Tippy tooltip docs of what I'm talking about: 
![image](https://user-images.githubusercontent.com/2730609/137529052-4cd0f4f5-2080-4eca-b226-1ee78dce6b6f.png)

https://atomiks.github.io/tippyjs/#interactivity

Hope this request makes sense. Thanks!

**Comment by adidahiya** [comment]
Why not use a Popover instead, in this case?

**Comment by tnrich** [comment]
@adidahiya because the interaction is different, with a blueprint popover you'll have to click to have the tooltip show up. In the tippy example I sent, you don't have to click.

**Comment by adidahiya** [comment, marked as answer]
Just use `interactionKind="hover"`:

![image](https://user-images.githubusercontent.com/723999/137770542-36791c94-0064-4476-a870-9bdb5c5968db.png)

https://codesandbox.io/s/blueprint-sandbox-forked-ex2bn?file=/src/Popover2Example.tsx
